samples/nft_deployment/README.md

```diff
@@ -0,0 +1,291 @@
+  --admin \
+  -u idemixorg.example.com \
+  -e alice \
+  -r 140
+
+# bob
+idemixgen signerconfig \
+  --ca-input $IDEMIX_CRYTPO/peerOrganizations/idemixorg.example.com \
+  --output $IDEMIX_CRYTPO/peerOrganizations/org2.example.com/peers/bob.org2.example.com/extraids/idemix \
+  --admin \
+  -u idemixorg.example.com \
+  -e bob \
+  -r 150
+```
+
+TODO 
+
+### Add Idemix Org to channel
+
+1. Add idemixOrgs to network https://hyperledger-fabric.readthedocs.io/en/latest/idemix.html
+2. Update Channel configuration
+
+
+## Token Validation Chaincode
+
+We begin by building all components used in the nft sample.
+The Token SDK uses a Token Validation Chaincode with Fabric to translate Token Transaction into Fabric Transaction with RWSets.
+See more details in [TODO])().
+
+### Generate crypto material
+
+TODO: We need to generate crypto material for the Issuer, the Auditor, and the Token Owners.
+Issuer and Auditor are X509 certificates. For Token Owners, we need to setup an Idemix CA with curve `BN254`. 
+
+### Prepare public parameters
+
+We are using `tokengen` to create the public parameters used by the Token Validation Chaincode in our nft sample.
+
+tokengen gen fabtoken
+
+```bash
+go install github.com/hyperledger-labs/fabric-token-sdk/cmd/tokengen
+tokengen gen dlog \ 
+  --auditors $IDEMIX_CRYTPO/peerOrganizations/org1.example.com/peers/auditor.org1.example/com/extraid/idemix/msp \ 
+  --issuers $IDEMIX_CRYTPO/peerOrganizations/org1.example.com/peers/issuer.org1.example/com/extraid/idemix/msp\
+  --cc
+
+cat zkatdlog_pp.json | base64
+```
+
+Copy the base64-encoded params into `$FTS_PATH/token/services/network/tcc/params.go`
+
+```go
+package tcc
+
+const Params = `>>>BASE64_STRING<<<`
+```
+
+
+### Build
+
+Once you have updated the `params.go`, you can compile TCC and package it using a `Dockerfile`.
+
+```bash
+go build $FTS_PATH/token/services/network/tcc/main
+```
+
+TODO
+
+### Deploy
+
+Run it as chaincode-as-a-service
+
+TODO
+
+## FSC Node
+
+The Token SDK builds on top of the Fabric Smart Client (FSC). The business logic implemented using the View API of the Fabric Smart Client is executed by so called FSC nodes. Every participant (i.e., Alice, Bob, the Issuer, and the Auditor) in the nft sample hosts a FSC node. 
+
+### Create crypto material
+
+TODO add
+
+- FSC node crypto material
+  - p2p
+  - grpc
+  - tls
+  - wallets
+
+#### FTS Wallet crypto material (idemix)
+
+```bash
+
+go install github.com/hyperledger-labs/fabric-smart-client/cmd/fsccli
+
+fsccli cryptogen generate \
+  --config $TOKEN_CRYPTO/default-testchannel-zkat/issuer/issuers/crypto-config.yaml \
+  --output $TOKEN_CRYPTO/default-testchannel-zkat/issuer/issuers
+
+fsccli cryptogen generate \
+  --config $TOKEN_CRYPTO/default-testchannel-zkat/auditor/auditors/crypto-config.yaml \
+  --output $TOKEN_CRYPTO/default-testchannel-zkat/auditor/auditors
+ 
+idemixgen signerconfig \
```

Comment:
We need to generate first a dedicated Issuer CA using `idemixgen ca-keygen`. This must happen before the generation of the public parameters because in those we need to include also the Issuer's PK.